### DISCUSSION TITLE
Rearrange stack order within a group in Group phase

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#4595] [Ideas]
Issue - Enhancement
In a retrospective, when creating a group, the latest card added to the group is at the top of the stack. It's the most visible card. Sometimes, that card is a good representation of the stack. Other times, it's more of an outlier.

One of our investors/users requested the option to change the stack order, so that a card that's on top and most visible could be a card that's more representative.

Acceptance Criteria (optional)
Users can change the stack order of cards within a group.

**Comment by gcrickman** [comment]
This [user](https://app.hubspot.com/contacts/3888472/company/3220410711) 🔒  mentioned:

> During the process of grouping cards, it's kind of hard to choose which one is the header or the cover. It would nice to be able to choose the first one visible for a group
